tutorials/migrating/upgrading_to_godot_4.5.rst

```diff
@@ -231,6 +231,25 @@ Core
     the shortest arc between the two input vectors. Previously, it would return incorrect values for certain inputs
     (`GH-107618`_).
 
+Navigation
+~~~~~~~~~~
+
+.. note::
+
+    NavigationServer regions now by default update async using threads to improve performance.
+    This can cause additional delay in the update due to thread synchronisation.
+    The async region update can be toggled with the ``navigation/world/region_use_async_iterations`` project setting.
+
+.. note::
+    NavigationServer navmesh merging has changed processing order as regions now merge and
+    cache internal navmesh first before remaining free edges are merged by the navigation map.
+    If a project had navigation map synchronisation errors before it might now have shifted
+    affected edges making already existing errors in a layout more noticeable in the pathfinding.
```

Comment:
```suggestion
    The merging of navmeshes in the NavigationServer has changed processing order. Regions now merge and cache
    internal navmeshes first, then the remaining free edges are merged by the navigation map.
    If a project had navigation map synchronisation errors before, it might now have shifted
    affected edges, making already existing errors in a layout more noticeable in the pathfinding.
```